assets/css/index.css

```diff
@@ -20,3 +20,23 @@ body {
   margin: 0 auto;
   max-width: 1160px;
 }
+
+a {
+  color: #28646e;
+  text-decoration: underline;
+}
+
+button {
+  background-color: #28646e;
+  color: white;
+  padding: 5px 10px;
+  margin: 3px;
+  border-radius: 4px;
+}
```

Comment:
This will conflict `likecoin-wallet-connector` style

Reply:
Should I add class on every button?

Reply:
Yes, should specify class for each button to prevent styling issue

Reply:
Fixed. BTW `sed` is awesome `sed -i 's/<button/<button class="button"/g' components/*`